include/superior_mysqlpp/logging.hpp

```diff
@@ -618,53 +637,61 @@ namespace SuperiorMySqlpp {
                 std::cerr << stringify("Pool [", id, "]: Health care job unknown ERROR!\n") << std::flush;
             }
 
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementCycleStart(std::uint_fast64_t /*poolId*/) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementChangeDetected(std::uint_fast64_t /*poolId*/) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementCheckError(std::uint_fast64_t /*poolId*/, std::exception&) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementCheckError(std::uint_fast64_t /*poolId*/) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementCycleEnd(std::uint_fast64_t /*poolId*/) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementStopped(std::uint_fast64_t /*poolId*/)const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementError(std::uint_fast64_t /*poolId*/, std::exception&) const noexcept override {}
+            [[deprecated]] virtual void logSharedPtrPoolDnsAwarePoolManagementError(std::uint_fast64_t /*poolId*/) const noexcept override {}
 
-            virtual void logSharedPtrPoolDnsAwarePoolManagementCycleStart(std::uint_fast64_t poolId) const noexcept override
+            virtual void logSharedPtrPoolDnsAwarePoolManagementCycleStart(std::uint_fast64_t poolId, const std::string &hostname) const noexcept override
             {
                 SpinGuard guard{lock};
-                std::cerr << stringify("Pool [", poolId, "]: DnsAwarePoolManagement: Cycle start.\n") << std::flush;
+                std::cerr << stringify("Pool [", poolId, ", hostname=", hostname, "]: DnsAwarePoolManagement: Cycle start.\n") << std::flush;
```

Comment:
mozna je to out-of-scope, ale co bys rikal na to, ze bychom se vyhnuli tomu vytvareni temporary stringu a predelali to na primy vypis do stringu? Ten stringify_helper je genericky a sel by pouzit, takze by to chtelo jen pridat funkci `toStream` co bere stream a varargs a vraci dany stream (abys mohl udelat ten flush)

Reply:
Je to dobry napad, ale myslim si ze je to out-of-scope, bylo by to nutne zmenit u vsech metod v loggerech, coz se nevztahuje k tomuhle tasku. BTW, nevis jaky duvod ma pouziti `'\n' << std::flush` misto `std::endl`?

Reply:
I don't see any good reason.

Would you make a ticket for future refactoring of that out-of-scope feature?

Reply:
Yep